packages/fxa-auth-server/scripts/convert-customers-to-stripe-automatic-tax.ts

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import program from 'commander';
+
+import { setupProcessingTaskObjects } from '../lib/payments/processing-tasks-setup';
+import { StripeAutomaticTaxConverter } from './convert-customers-to-stripe-automatic-tax/convert-customers-to-stripe-automatic-tax';
+
+const pckg = require('../package.json');
+
+const parseDryRun = (dryRun: boolean | string) => {
+  return `${dryRun}`.toLowerCase() !== 'false';
+};
+
+const parseBatchSize = (batchSize: string | number) => {
+  return parseInt(batchSize.toString(), 10);
+};
+
+const parseOutputFile = (outputFile: string) => {
+  return outputFile;
+};
```

Comment:
A no-op?

Reply:
I kept this for:
1. Consistency
2. The type conversion from any to string
3. Path joining in https://mozilla-hub.atlassian.net/browse/FXA-6581